specification/translation/Azure.AI.DocumentTranslation/models.tsp

```diff
@@ -215,87 +190,6 @@ model Glossary {
   storageSource?: StorageSource;
 }
 
-@doc("""
-  Contains unified error information used for HTTP responses across any Cognitive
-  Service. Instances
-  can be created either through
-  Microsoft.CloudAI.Containers.HttpStatusExceptionV2 or by returning it directly
-  from
-  a controller.
-  """)
-@error
-model TranslationErrorResponse {
-  @doc("""
-    This contains an outer error with error code, message, details, target and an
-    inner error with more descriptive details.
-    """)
-  error?: TranslationError;
-}
-
-@doc("""
-  This contains an outer error with error code, message, details, target and an
-  inner error with more descriptive details.
-  """)
-model TranslationError {
-  @doc("Enums containing high level error codes.")
-  code: TranslationErrorCode;
-
-  @doc("Gets high level error message.")
-  message: string;
-
-  @doc("""
-    Gets the source of the error. 
-    For example it would be \"documents\" or
-    \"document id\" in case of invalid document.
-    """)
-  @visibility("read")
-  target?: string;
-
-  @doc("""
-    New Inner Error format which conforms to Cognitive Services API Guidelines
-    which is available at
-    https://microsoft.sharepoint.com/%3Aw%3A/t/CognitiveServicesPMO/EUoytcrjuJdKpeOKIK_QRC8BPtUYQpKBi8JsWyeDMRsWlQ?e=CPq8ow.
-    This
-    contains required properties ErrorCode, message and optional properties target,
-    details(key value pair), inner error(this can be nested).
-    """)
-  innerError?: InnerTranslationError;
-}
-
-@doc("""
-  New Inner Error format which conforms to Cognitive Services API Guidelines
-  which is available at
-  https://microsoft.sharepoint.com/%3Aw%3A/t/CognitiveServicesPMO/EUoytcrjuJdKpeOKIK_QRC8BPtUYQpKBi8JsWyeDMRsWlQ?e=CPq8ow.
-  This
-  contains required properties ErrorCode, message and optional properties target,
-  details(key value pair), inner error(this can be nested).
-  """)
-model InnerTranslationError {
```

Comment:
Looks like your inner error type has more properties than the one in the Foundations library

Reply:
Look at message for example

Reply:
You are right, we cannot have this breaking change since its not the same model and we have already GA'd this API version. Will plan to address this in the next APIView.
So reverted this change.